egs/librispeech/s5/run.sh

```diff
@@ -327,6 +327,10 @@ steps/train_quick.sh --cmd "$train_cmd" \
   done
 )&
 
+# this does some data-cleaning. The cleaned data should be useful when we add
+# the neural net and chain systems.
+local/run_cleanup_segmentation.sh
+
```

Comment:
While you are doing this you might also want to update `run.sh` to include commands for running the nnet3 systems (xent and chain).